lib/smart_answer_flows/overseas-passports/outcomes/getting_your_passport/_collect_in_person.govspeak.erb

```diff
@@ -0,0 +1,2 @@
+
+You must collect in person.  If you’re unable to, someone else can go on your behalf.
```

Comment:
Very minor: The two spaces at the beginning of the 2nd sentence seems inconsistent with most of the other content in the templates.